Agents/ForecastingAgent/README.md

```diff
@@ -0,0 +1,127 @@
+# Forecasting Agent
+
+<span style="color:red">Tests are currently still excluded.</span>
+
+# 1. Setup
+
+This section specifies the minimum requirements to run the agent. 
+
+&nbsp;
+## 1.1 Prerequisites
+
+
+### **1) Setting up a virtual environment setup**
+
+It is highly recommended to use a virtual environment for this project. The virtual environment can be created as follows (`python` command might need to be replaced with `py` depending on whether Python is specified in system's `PATH` variables):
+
+`(Windows)`
+```cmd
+$ python -m venv forecasting_venv
+$ forecasting_venv\Scripts\activate.bat
+(forecasting_venv) $
+```
+The above commands will create and activate the virtual environment `forecasting_venv` in the current directory.
+
+### **2) Installation of required packages**
+
+Install the `forecasting` project including all required packages listed in `setup.py`:
+
+`(Windows)`
+```
+python -m pip install --upgrade pip  
+python -m pip install -e . 
+```
+If you use later a model pretrained with 'darts', conflicts can occur while loading the model, if your version differs from the version with which the model was trained.
+
+### **3) Instantiated knowledge graph with time series**
+
+In order to forecast a time series, this series has to be instantiated using  [this Time Series Client](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries) beforehand.    
+
+### **4) Endpoints**
+
+Set your postgres database and blazegraph endpoints in your properties [file](./resources/timeseries.properties). 
+
+
+&nbsp;
+# 2. Using the Agent
+## General workflow
+- [Agent uml](https://lucid.app/lucidchart/def34dba-537c-48c7-9fa4-89bda55b4dc5/edit?viewport_loc=-3263%2C-197%2C3677%2C1765%2C0_0&invitationId=inv_1ed2a56a-16f0-4884-a5cb-a5aa69daba1e)
```

Comment:
Embedding is not possible because Iframe is not supported. I guess link is better than downloaded image

Reply:
Hi, my two cents here as I noticed you are using Lucidchart - you may want to publish the UML as png and insert the link here. It will render the image when you open the README on a webpage. An example line to add in README:
```![Agent UML](https://lucid.app/publicSegments/view/uuid_generated_when_publishing/image.png)```

Note tho the image is NOT version controlled by git, i.e. it always reflects the latest changes in Lucidchart when you load the README (without changing the committed link).